crates/extensions/c8y_mapper_ext/src/entity_cache.rs

```diff
@@ -170,8 +169,8 @@ impl EntityCache {
             external_id: Some(external_id.clone()),
             r#type: entity.r#type,
             parent,
-            other: entity.other,
-            twin_data: Map::new(),
+            other: entity.other.clone(),
```

Comment:
Now that this `other` field represents initial twin data, this could be merged with the `twin_data` field. I can't think of any benefits in keeping the **initial** twin data cached separately from the **current** twin data forever.

Reply:
I have always been confused by these two fields, `other` vs `twin_data`, and I would be happy to have one removed if there are no impacts from a user perspective.

By the way, why is `other` tagged with as `serde(flatten)` and `twin_data` as `serde(skip)`?

Reply:
Resolved by 36b07fb